generator/.env.example

```diff
@@ -0,0 +1,3 @@
+PUBLIC_DEVELOPMENT=false
+PUBLIC_API_BASE_URL_LOCAL=http://localhost:8081
+API_BASE_URL_PROD=https://api.tags.ioxio.dev
```

Comment:
No, this is not how you configure which one you use. You set one variable, it will be `PUBLIC_API_BASE_URL=http://localhost:8081`, and in the deployed environments we will override it.